apps/asap-server/src/controllers/calendars.ts

```diff
@@ -27,11 +27,33 @@ export default class Calendars implements CalendarController {
       items: calendars.map(parseCalendar),
     };
   }
+
+  async getSyncToken(calendarId: string): Promise<string | undefined> {
```

Comment:
if you feel like it you can change it to `fetchRaw` as we are going to need to be able to fetch more than just the sync token in the near future